st2common/st2common/util/config_loader.py

```diff
@@ -169,6 +169,11 @@ def _assign_dynamic_config_values(self, schema, config, parent_keys=None):
                 is_jinja_expression = jinja_utils.is_jinja_expression(
                     value=config_item_value
                 )
+                if "decrypt_kv" in str(config_item_value):
+                    LOG.audit("User %s is decrypting the value %s from the config within pack %s", self.user,
+                              config_item_value, self.pack_name,
+                    extra = {"user": self.user, "key_name": config_item_key, "pack_name": self.pack_name,
+                             "config_item_value": config_item_value, "operation": "pack_config_value_decrypt"})
```

Comment:
Considering you tested it manually, could you provide a real example of a logged result from this construct?

Reply:
![image](https://user-images.githubusercontent.com/40653159/160833980-23d93c4b-15b4-400c-8698-99a1d5d025b1.png)

30 12:25:32,730 AUDIT [-] User st2admin is decrypting the value {{ st2kv.system.nso.f5_VIP_address | decrypt_kv }} from the config within pack cbanso_demo (user='st2admin',key_name='f5_VIP_address',pack_name='cbanso_demo',config_item_value='{{ st2kv.system.nso.f5_VIP_address | decrypt_kv }}',operation='pack_config_value_decrypt')

2022-03-